data/strings/strings.txt

```diff
@@ -5849,8 +5860,8 @@
     ko = 우리는 Organic Maps. Lite에서 Organic Maps로 다운로드된 지도를 전송하고 있습니다. 이는 몇 분 정도 걸릴 수 있습니다.
     nb = Vi overfører dine nedlastede kart fra Organic Maps Lite til Organic Maps. Det kan ta noen minutter.
     pl = Przenosimy twoje pobrane mapy z Organic Maps Lite do Organic Maps. Może to zająć kilka minut.
-    pt = Estamos a transferir os seus mapas descarregados da Organic Maps Lite para a Organic Maps. Isto pode demorar alguns minutos.
-    pt-BR = Estamos transferindo os seus mapas baixados do Organic Maps Lite para o Organic Maps. Isto pode demorar alguns minutos.
```

Comment:
[move_lite_maps_to_pro]
[move_lite_maps_to_pro_ok]
[move_lite_maps_to_pro_failed]
These translated blocks should be completely removed, they are not used anymore for a long time.